src/routes/game/GameView.vue

```diff
@@ -9,7 +9,7 @@
     <template v-else>
       <div id="game-menu-wrapper" class="d-flex flex-column flex-sm-row align-center" :style="menuWrapperStyle">
         <SpectatorListMenu :spectating-users="spectatingUsers" :vuetify-display="$vuetify" />
-        <GameMenu :is-spectating="isSpectating" />
+        <GameMenu :is-spectating="isSpectating" @handle-error="handleError" />
```

Comment:
Added error handling to the `GameMenu` so we can see snackbar errors when you request an illegal stalemate (e.g. b/c you requested one already this turn)